set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     BSADBUARVDUJVEVFVGDALHJVHUNVI $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Real subtraction
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c -R $.
+
+  $( Real number subtraction. $)
+  crsub $a class -R $.
```

Comment:
As @digama0 sugested, the prefix "re" should be used for this real number theorems and definitions. This also conserns the label above (crsunb -> cresub), the following definition (df-rsub -> df-resub) and most of the related theorems (rsubval -> resubval).

Reply:
There are still some theorem labels prefixed with "r" instead of "re": ~rnegcl, ~rsubcl

Reply:
renegcl and resubcl already exist, the name conflicts

Reply:
I cannot see the changes because github is still down, but since your novelty is to use your flavor of "real negation/substraction" instead of negation/substraction, maybe `rernegcl / rersubcl`.

Reply:
OK now